include/gps.hpp

```diff
@@ -44,24 +26,25 @@ struct GpsDate {
 struct GpsPosition {
     bool fix;
     uint8_t fixquality;
-    GpsFloat longitude;
+    nmea_float_t longitude;
```

Comment:
przydałby się jakiś komentarz czy to jest w stopniach czy dec, podobnie do latitude

Reply:
takie informacje trzyma sie w type systemie ale to wymaga troche wiecej roboty, mam to z tyłu głowy